setup.cfg

```diff
@@ -1,24 +1,125 @@
+[metadata]
+name = woodwork
+version = 0.14.0
+author = Alteryx Inc.
+license = BSD 3-clause
+long_description = file: READEME.md
+url = https://github.com/alteryx/woodwork/
+long_description_content_type = 'text/markdown'
```

Comment:
I don't know for sure if it matters, but we had trouble with this in Featuretools and ended up specifying this without any quotes: https://github.com/alteryx/featuretools/blob/aaa84e3634988b2e6d2c528ff240a397655024d7/setup.cfg#L31

Reply:
updated